be/src/vec/olap/vcollect_iterator.cpp

```diff
@@ -285,11 +285,6 @@ Status VCollectIterator::_topn_next(Block* block) {
 
             auto col_name = block->get_names()[first_sort_column_idx];
 
-            // filter block
-            RETURN_IF_ERROR(VExprContext::filter_block(
```

Comment:
Do you check this with hangyu? it is useless?

Reply:
Checked with them, can be replaced